mkosi/config.py

```diff
@@ -2069,12 +2069,17 @@ def sandbox(
         options: Sequence[PathString] = (),
         setup: Sequence[PathString] = (),
     ) -> AbstractContextManager[list[PathString]]:
-        opt: list[PathString] = [
-            *options,
-            *(["--ro-bind", str(p), "/proxy.cacert"] if (p := self.proxy_peer_certificate) else []),
-            *(["--ro-bind", str(p), "/proxy.clientcert"] if (p := self.proxy_client_certificate) else []),
-            *(["--ro-bind", str(p), "/proxy.clientkey"] if (p := self.proxy_client_key) else []),
-        ]
+        opt: list[PathString] = [*options]
+        if not relaxed:
+            opt += [
+                *(["--ro-bind", str(p), "/proxy.cacert"] if (p := self.proxy_peer_certificate) else []),
+                *(
+                    ["--ro-bind", str(p), "/proxy.clientcert"]
+                    if (p := self.proxy_client_certificate)
+                    else []
+                ),
+                *(["--ro-bind", str(p), "/proxy.clientkey"] if (p := self.proxy_client_key) else []),
+            ]
```

Comment:
The additional level of indentation makes this less readable, because the middle option gets torn apart apart.

It could be more compactly written as
 ```suggestion
        opt: list[PathString] = [*options]
        if not relaxed:
            if p := self.proxy_peer_certificate:
                opt += ["--ro-bind", os.fspath(p), "/proxy.cacert"]
            if p := self.proxy_client_certificate:
                opt += ["--ro-bind", os.fspath(p), "/proxy.clientcert"]
            if p := self.proxy_client_key:
                opt += ["--ro-bind", os.fspath(p), "/proxy.clientkey"]
```
or just
```suggestion
        opt: list[PathString] = [*options]
        if not relaxed:
            opt += [
                *(["--ro-bind", os.fspath(p), "/proxy.cacert"] if (p := self.proxy_peer_certificate) else []),
                *(["--ro-bind", os.fspath(p), "/proxy.clientcert"] if (p := self.proxy_client_certificate) else []),
                *(["--ro-bind", os.fspath(p), "/proxy.clientkey"] if (p := self.proxy_client_key) else []),
            ]  # fmt: skip
```